### PR TITLE
Grammatically updated the readme file.

### PR DESCRIPTION
**Pull request by professorabhay**
Screenshot 2022-11-09 182401

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,9 +19,9 @@ Read our [Download and Installation](https://www.xwiki.org/xwiki/bin/view/Downlo
 Read our [Release Notes](https://www.xwiki.org/xwiki/bin/view/ReleaseNotes/).
 
 ## Tools
-* [Continuous Integration](https://ci.xwiki.org/) setup launches a build for each commit
-* [Issue Tracker](https://jira.xwiki.org/browse/XWIKI) if you want to report an issue
-* [Development Flow](https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#HGeneralDevelopmentFlow) to see the full list of tools we use to build XWiki
+* [Continuous Integration](https://ci.xwiki.org/) setup launches a build for each commit.
```
Comment (vmassol) [inline README.md:22]: I don't think we want the extra space at the end.

File: README.md [hunk 2]
```diff
@@ -19,9 +19,9 @@ Read our [Download and Installation](https://www.xwiki.org/xwiki/bin/view/Downlo
 Read our [Release Notes](https://www.xwiki.org/xwiki/bin/view/ReleaseNotes/).
 
 ## Tools
-* [Continuous Integration](https://ci.xwiki.org/) setup launches a build for each commit
-* [Issue Tracker](https://jira.xwiki.org/browse/XWIKI) if you want to report an issue
-* [Development Flow](https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#HGeneralDevelopmentFlow) to see the full list of tools we use to build XWiki
+* [Continuous Integration](https://ci.xwiki.org/) setup launches a build for each commit.   
+* [Issue Tracker](https://jira.xwiki.org/browse/XWIKI), if you want to report an issue.
+* [Development Flow](https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#HGeneralDevelopmentFlow) to see the full list of tools we use to build "XWiki".
```
Comment (vmassol) [inline README.md:24]: I don't think we need quotes here. That would be such a pain to do this wherever XWiki is mentioned ;)

**Comment by vmassol** [comment]
Thanks for the PR! I've left 2 comments to address before the PR  can be applied.

File: README.md (hunk 2 above)
Comment (professorabhay) [reply README.md:24]: Ok,
But I think we don't need to do this everywhere but at that line, it looks perfect.

Comment (professorabhay) [reply README.md:24]: What about full stop and comma ? 
I can remove quotes if you like to update 
( . Or , ) .

Comment (vmassol) [reply README.md:24]: Maybe we could write `to build the XWiki software` if you think that's better?

Comment (vmassol) [reply README.md:24]: Full stop and commas look ok.

Comment (professorabhay) [reply README.md:24]: Hmm....it's also looks good. 
We can do this.

Comment (professorabhay) [reply README.md:24]: Ok 🆗
That's my very first open source contribution/pr
Starting my journey with xwiki.
I'll definitely going to contribute more valuable!!

**Comment by professorabhay** [comment]
> Thanks for the PR! I've left 2 comments to address before the PR  can be applied.

Hey, what's the update?

File: README.md [hunk 3]
```diff
@@ -19,9 +19,9 @@ Read our [Download and Installation](https://www.xwiki.org/xwiki/bin/view/Downlo
 Read our [Release Notes](https://www.xwiki.org/xwiki/bin/view/ReleaseNotes/).
 
 ## Tools
-* [Continuous Integration](https://ci.xwiki.org/) setup launches a build for each commit
-* [Issue Tracker](https://jira.xwiki.org/browse/XWIKI) if you want to report an issue
-* [Development Flow](https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#HGeneralDevelopmentFlow) to see the full list of tools we use to build XWiki
+* [Continuous Integration](https://ci.xwiki.org/) setup launches a build for each commit.   
+* [Issue Tracker](https://jira.xwiki.org/browse/XWIKI), if you want to report an issue.
+* [Development Flow](https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#HGeneralDevelopmentFlow) to see the full list of tools we use to build the   XWiki software.
```
Comment (vmassol) [inline README.md:24]: It seems there's an extra space after `the`.

**Review by acotiuga — CHANGES_REQUESTED** [review]

**Pull request merged**